schema/Makefile

```diff
@@ -0,0 +1,10 @@
+
+default: help
+
+help:
+	@echo "Usage: make <target>"
+	@echo
+	@echo " * 'fmt' - format the json with indentation"
+
+fmt:
+	for i in *.json ; do jq --indent 4 -M . "$${i}" > xx && cat xx > "$${i}" && rm xx ; done
```

Comment:
There would be no `*.json` churn if you dropped the `--indent 4` and stuck with jq's default two-space indent.